lib/twitter.rb

```diff
@@ -41,6 +41,7 @@ def tweet(clinic)
     else
       @twitter.update(text)
     end
+  end
```

Comment:
This file should be stay the same, moving the `end` to here may change the behavior.